Makefile

```diff
@@ -6,14 +6,14 @@
 ################################################################################
 
 ## Docker Build Versions
-DOCKER_BUILD_IMAGE = golang:1.14.6
-DOCKER_BASE_IMAGE = alpine:3.12
+DOCKER_BUILD_IMAGE = golang:1.16.2
+DOCKER_BASE_IMAGE = alpine:3.13.2
 
 ## Tool Versions
 TERRAFORM_VERSION=0.12.29
```

Comment:
Theoretically we can support terraform 0.13.5 right? Should we give it a go or not in this PR?

Reply:
Yep, that right. I was planning on doing that as a follow-up due to this already having a bunch of changes.